app/views/admin/works/_combined_audio_derivatives.html.erb

```diff
@@ -8,10 +8,34 @@
     <% else %>
       <% if view.combined_audio_fingerprint.present? %>
         <% if view.derivatives_up_to_date? %>
-          <p class="alert-success p-3">
-            <i class="fa fa fa-thumbs-up pr-1" aria-hidden="true"></i>
-            The combined audio derivatives, created from <%= view.work_available_members_count %> published audio segments, are up to date. Their fingerprint is <code class="border"><%= view.combined_audio_fingerprint%></code>.</p>
-          </p>
+          <% if model&.oral_history_content&.combined_audio_derivatives_job_status_changed_at.nil? %>
+            <p class="alert-success p-3">
+                <i class="fa fa fa-thumbs-up pr-1" aria-hidden="true"></i>
+                  The combined audio derivatives, created from <%= view.work_available_members_count %> published audio segments, are up to date. Their fingerprint is <code class="border"><%= view.combined_audio_fingerprint%></code>.
+            </p>
+          <% end %>
+          <% if model&.oral_history_content&.queued?  %>
+            <p class="alert-info p-3">
+              The combined audio derivatives will be recreated soon. The job was added to the queue <%=view.time_since_job_status_change %>.
+            </p>
+          <% end %>
+          <% if model&.oral_history_content&.started?  %>
+            <p class="alert-info p-3">
+              The combined audio derivatives are being recreated. The job started <%=view.time_since_job_status_change %>.
+            </p>
+          <% end %>
+          <% if model&.oral_history_content&.failed?  %>
+            <p class="alert-danger p-3">
+              <i class="fa fa fa-thumbs-down pr-1" aria-hidden="true"></i>
+              The combined audio derivatives were not recreated. The job failed <%=view.time_since_job_status_change %>.
+            </p>
+          <% end %>
+          <% if model&.oral_history_content&.done?  %>
+            <p class="alert-success p-3">
+              <i class="fa fa fa-thumbs-up pr-1" aria-hidden="true"></i>
+                The combined audio derivatives, created <%=view.time_since_job_status_change %> from <%= view.work_available_members_count %> published audio segments, are up to date. Their fingerprint is <code class="border"><%= view.combined_audio_fingerprint%></code>.
+            </p>
+          <% end %>
```

Comment:
This is way too much logic/code for this staff facing feature. The more branches you have, the more opportunity for bugs and such, the more to maintain. 

I think it should just be:  "Combined audio derivative creation status: #{status}, since #{formatted status changed at time}". Maybe with a red color for error, that's about it. 

Not sure if it should be there at all for `done` (to be changed to `succeeded`) -- if it is, it means it will basically be there forever? Not sure we need it for `done` at all, or alternately only if `done` is in the last X hours (24? I dunno). But I think for "done" it might be okay just that... you see the derivative is there now, that's all the feedback you need?

Reply:
Yup -- drastically simplified. Good call.